tests/functional/pv/pvc_snapshot/test_snapshot_restore_with_different_access_mode.py

```diff
@@ -57,6 +59,7 @@ def test_snapshot_restore_with_different_access_mode(
                 constants.VOLUME_MODE_FILESYSTEM: [
                     constants.ACCESS_MODE_RWX,
                     constants.ACCESS_MODE_RWO,
+                    constants.ACCESS_MODE_ROX,
```

Comment:
Skip this access mode in the check "Verify md5sum" at the end of this testcase.

Reply:
Done